concordia/templates/home.html

```diff
@@ -51,49 +51,48 @@ <h4 class="offwhite-text mt-half">REVIEW</h4>
         <div class="col-12 mb-half">
             <h2 class="px-default px-md-0 ml-md-default">Campaigns: <small>Choose what to transcribe</small></h2>
         </div>
+
+
+
         <div class="d-flex flex-wrap col-12 justify-content-around">
-            <div class="col-12 col-md-4 mb-default text-center">
-                <img class="img-home-contribute d-block mx-auto img-fluid" src="{% static 'img/rosa-parks-campaign.svg' %}"
-                    alt="Rosa Parks Papers">
-                <h3 class="mt-half">Rosa Parks Papers</h3>
-                <a class="btn btn-outline-primary" href="#" role="button">Get Started!</a>
-            </div>
-            <div class="col-12 col-md-4 mb-default text-center border-left-home-contribute">
-                <img class="img-home-contribute d-block mx-auto img-fluid" src="{% static 'img/branch-rickey-campaign.svg' %}"
-                    alt="Branch Rickey Papers">
-                <h3 class="mt-half">Branch Rickey Papers</h3>
-                <a class="btn btn-outline-primary" href="#" role="button">Get Started!</a>
-            </div>
-            <div class="col-12 col-md-4 mb-default text-center border-left-home-contribute">
-                <img class="img-home-contribute d-block mx-auto img-fluid" src="{% static 'img/clara-barton-campaign.svg' %}"
-                    alt="Clara Barton Papers">
-                <h3 class="mt-half">Clara Barton Papers</h3>
-                <a class="btn btn-outline-primary" href="#" role="button">Get Started!</a>
-            </div>
+            {% for campaign in campaigns %}
+            {% if forloop.counter == 1 or forloop.counter == column_count|add:"1" %}
+            {% if forloop.counter > 1 %}
         </div>
+        {% endif %}
         <div class="d-flex flex-wrap col-12 justify-content-around">
-            <div class="col-12 col-md-4 mb-default text-center">
-                <img class="img-home-contribute d-block mx-auto img-fluid" src="{% static 'img/AbrahamLincoln.svg' %}"
-                    alt="Abraham Lincoln Papers">
-                <h3 class="mt-half">Abraham Lincoln Papers</h3>
-                <a class="btn btn-outline-primary" href="#" role="button">Get Started!</a>
-            </div>
-            <div class="col-12 col-md-4 mb-default text-center border-left-home-contribute">
-                <img class="img-home-contribute d-block mx-auto img-fluid" src="{% static 'img/MaryChurchTerrell.svg' %}"
-                    alt="Mary Church Terrell Papers">
-                <h3 class="mt-half">Mary Church Terrell Papers</h3>
-                <a class="btn btn-outline-primary" href="#" role="button">Get Started!</a>
+            {% comment %}
+            FIXME: col-md-6 when there are 4 campaigns to display, col-md-4 when there are 6
```

Comment:
I realize you inherited this but it really looks to me like we could simplify all of this to a simply using flexbox and not trying to abuse the column layout classes to set item widths

Reply:
https://github.com/LibraryOfCongress/concordia/pull/310 has my experiment in this direction